anesthetic/plot.py

```diff
@@ -131,57 +128,66 @@ def make_2d_axes(params, **kwargs):
         xparams, yparams = params
     else:
         xparams = yparams = params
+
+    upper = kwargs.pop('upper', True)
+    lower = kwargs.pop('lower', True)
+    diagonal = kwargs.pop('diagonal', True)
```

Comment:
Are these defaults inconsistent with the default for `types` in `samples.plot_2d`?
```python
types = kwargs.pop('types', {'diagonal': 'kde', 'lower': 'kde'})
```

Reply:
They are, but the ``make_2d_plots`` is a separate set of code, so they don't need to be the same. I think I might prefer to have the default include the scatter plot above the diagonal, since it wastes less space and does provide orthogonal (and unsmoothed) information.

Reply:
I agree, but maybe it would make sense to change `types` to behave in the same way? 
At the very least it would make the options of `upper` and of `scatter` known to the user and should they wish to exclude them the docs should be straight forward in telling how to turn them off.

Reply:
I've now changed this default in the latest push